tests/mq_llm_engine/test_error_handling.py

```diff
@@ -59,15 +59,7 @@ async def test_evil_forward(tmp_socket):
         await asyncio.sleep(2.0)
         await client.check_health()
 
-        # Throws an error in first forward pass.
-        with pytest.raises(RAISED_ERROR):
-            async for _ in client.generate(prompt="Hello my name is",
-                                           sampling_params=SamplingParams(),
-                                           request_id=uuid.uuid4()):
-                pass
-        assert client.errored
-
-        # Engine is errored, should get ENGINE_DEAD_ERROR.
+        # Throws an error that should get ENGINE_DEAD_ERROR.
```

Comment:
We could also check a "batch" of requests here, like
```
def do_generate(client):
    async for _ in client.generate(prompt="Hello my name is",
                                   sampling_params=SamplingParams(),
                                   request_id=uuid.uuid4()):
        pass

...
# (in this test)
tasks = [asyncio.create_task(do_generate(client)) for _ in range(10)]

# Check that every `task` in `tasks` failed with `MQEngineDeadError`
```

That should test that we don't get the big spew of stack traces, since every request will raise an error type that doesn't log the stack trace